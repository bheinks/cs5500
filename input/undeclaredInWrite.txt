(*
	Undeclared identifier in write
*)

program p;

var bojack, todd : integer;

begin
  write(bojack, todd);
  write(princessCarolyn)
end.

