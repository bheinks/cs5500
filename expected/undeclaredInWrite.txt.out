

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding bojack to symbol table with type INTEGER

+++ Adding todd to symbol table with type INTEGER
Line 11: Undefined identifier
